src/pages/cadastrar-demanda/cadastrar-demanda.css

```
@@ -0,0 +1,115 @@
+
+.container{
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+.content{
+    background-color: #E8ECEF;
+    border-radius: 0;
+    align-self: center;
+    font-weight: 700;
+}
+.title {
+    position:relative;
+    color: #274c77;
+    font-size: 40px;
+    margin-bottom: 20px;
+    border-bottom: 1px dotted #716F6F; 
+    padding-top: 20px;
+    padding-bottom: 20px;
+}
+label{
+    display: inline-block;
+    color: #716F6F;
+    font-size: 20px;
```

Este tamanho de letra, como havíamos discutido no whatsapp, é demasiado grande. Se o remover, por agora, ficará com um bom tamanho, mas temos que tratar isso de forma transversal às telas em um css file geral.
```suggestion
    font-size: 20px;
```